### ISSUE TITLE
Swipe for next on mobile

### ISSUE THREAD

**Issue by dimmuboy** [dolce/iziModal#147]
Is there any way to go to next popup with touch on edge or much better with swipe on mobile devices? On small mobile screen popups are unusable without these features.

**Comment by groundnation** [comment]
+1!

**Comment by chupzzz** [comment]
Please check this solution for mobile navigation at #211

**Comment by dimmuboy** [comment]
Sorry but it's very uncomfortable :/

**Comment by chupzzz** [comment]
> Sorry but it's very uncomfortable :/

Yes, VERY! I don't like it too. But for now this is the only solution for mobiles.
